tools/jenkins/docker_run_tests.sh

```diff
@@ -43,6 +43,8 @@ chown $(whoami) $XDG_CACHE_HOME
 mkdir -p /var/local/git
 git clone --recursive /var/local/jenkins/grpc /var/local/git/grpc
 
+[ -e /post-git-setup.sh ] && /post-git-setup.sh
```

Comment:
I suggest setting an env variable to indicate the need to do post git setup and then run post git setup shell script from /var/local/git/grpc/.....